src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -56,6 +64,13 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
 
     private activeFoci: Focus[] | undefined;
 
+    private encryptMedia = false;
+    private _encryptionKeys = new Map<{ userId: string; deviceId: string }, string>();
```

Comment:
Docs please. What keys are they? For incoming or outgoing media? Where are they used?